MantidQt/CustomInterfaces/src/Indirect/JumpFit.cpp

```diff
@@ -226,12 +226,12 @@ void JumpFit::handleSampleInputReady(const QString &filename) {
   IAlgorithm_sptr scaleAlg = AlgorithmManager::Instance().create("Scale");
   scaleAlg->initialize();
   scaleAlg->setProperty("InputWorkspace", filename.toStdString());
-  scaleAlg->setProperty("OutputWorkspace", filename.toStdString());
+  scaleAlg->setProperty("OutputWorkspace", filename.toStdString() + "_HWHM");
```

Comment:
It would be good for maintainability if the output name with the "_HWHM" suffix was defined in only one place. Perhaps it could be cached as a class member, or returned by a utility function?